cfme/tests/cloud/test_cloud_timelines.py

```diff
@@ -157,7 +157,8 @@ def _power_off(self):
         return self.inst.mgmt.stop()
 
     def _power_off_power_on(self):
-        return self.inst.mgmt.restart()
+        self.inst.mgmt.stop()
+        return self.inst.mgmt.start()
```

Comment:
**Required** I think it be good to keep the resart functionality , add a new `_resart()` method perhaps.